### PR TITLE
Add script for deploying ERC20 rollups

### PR DESCRIPTION
**Pull request by gvladika**
RollupCreator supports deploying both ETH and ERC20 based rollups. This commit adds script which deploys erc20 based rollups (script will deploy basic ERC20 token to be used as fee token).

### CODE REVIEW

File: scripts/createERC20Rollup.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    await deployer.getAddress()
+  )
+  const feeToken = await factory.deployed()
+
+  return feeToken.address
+}
+
+async function main() {
+  const [deployer] = await ethers.getSigners()
+  const customFeeTokenAddress = await deployERC20Token(deployer)
```
Comment (gzeoneth) [inline scripts/createERC20Rollup.ts:21]: I think this should include the option to specify an existing token

Comment (gvladika) [reply scripts/createERC20Rollup.ts]: done

File: scripts/createERC20Rollup.ts [hunk 2]
```diff
@@ -18,7 +18,14 @@ async function deployERC20Token(deployer: Signer): Promise<string> {
...
   const [deployer] = await ethers.getSigners()
-  const customFeeTokenAddress = await deployERC20Token(deployer)
+
+  let customFeeTokenAddress = process.env.FEE_TOKEN_ADDRESS
+  if (!customFeeTokenAddress) {
+    console.log(
+      'FEE_TOKEN_ADDRESS env var not provided, deploying new ERC20 token'
+    )
+    customFeeTokenAddress = await deployERC20Token(deployer)
+  }
```
Comment (gzeoneth) [inline scripts/createERC20Rollup.ts:28]: might want to check if the supplied value is an valid address in `else`

**Review by gzeoneth — APPROVED** [review]
lgtm

File: scripts/createERC20Rollup.ts (hunk 2 above)
Comment (gvladika) [reply scripts/createERC20Rollup.ts:28]: done

**Pull request merged**